bip-0085.mediawiki

```diff
@@ -167,10 +196,18 @@ Words Table
 | 128 bits
 | 12'
 |-
+| 15 words
+| 160 bits
+| 15'
+|-
 | 18 words
 | 192 bits
 | 18'
 |-
+| 21 words
+| 224 bits
+| 21'
+|-
```

Comment:
Ok, these reflect the table in BIP39 "Generating the mnemonic" section.

Reply:
> @@ -95,7 +95,36 @@ OUTPUT
 
 * DRNG(80 bytes)=b78b1ee6b345eae6836c2d53d33c64cdaf9a696487be81b03e822dc84b3f1cd883d7559e53d175f243e4c349e822a957bbff9224bc5dde9492ef54e8a439f6bc8c7355b87a925a37ee405a7502991111
 
-==Reference Implementation==
+==Changelog==